Library/Formula/netperf.rb

```diff
@@ -11,6 +11,12 @@ class Netperf < Formula
     sha256 "4e0aa192670d1de341e77e0bd3af4b2cccc6a5b12a734e922e92c7b7d50f281d" => :mountain_lion
   end
 
+  # patch fixes demo mode which is required by a lot of software that depends on netperf
+  patch :p2 do
+    url "https://raw.githubusercontent.com/trelane/netperf-brew-patch/master/netlib.diff"
```

Comment:
in my case, flent, but in general anything which needs Time series database
output from netperf (so that it could be imported into RRDtool, MRTG, etc.



On Thu, Feb 11, 2016 at 2:45 PM, Mike McQuaid <notifications@github.com>
wrote:

> In Library/Formula/netperf.rb
> <https://github.com/Homebrew/homebrew/pull/49055#discussion_r52655759>:
>
> > @@ -11,6 +11,12 @@ class Netperf < Formula
> >      sha256 "4e0aa192670d1de341e77e0bd3af4b2cccc6a5b12a734e922e92c7b7d50f281d" => :mountain_lion
> >    end
> >
> > +  # patch fixes demo mode which is required by a lot of software that depends on netperf
> > +  patch :p2 do
> > +    url "https://raw.githubusercontent.com/trelane/netperf-brew-patch/master/netlib.diff"
>
> Can you elaborate on e.g. what software, when you emailed the developer
> and where.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/49055/files#r52655759>.
>